.github/workflows/release_tag.yml

```diff
@@ -128,14 +122,14 @@ jobs:
             export GITHUB_REPOSITORY=$(jq -r '.name' <<< "$i")
             export INPUT_PREVIOUSTAG=$(jq -r '.versions[0]' <<< "$i")
             export INPUT_CURRENTTAG=$(jq -r '.versions[1]' <<< "$i")
+            echo Running against $GITHUB_REPOSITORY
             echo -n "## " >> generator_output
             python main.py >> generator_output
             echo '' >> generator_output
           done
           echo "CHANGELOG_OUTPUT<<EOF" >> $GITHUB_ENV
           echo "$(cat generator_output)" >> $GITHUB_ENV
           echo "EOF" >> $GITHUB_ENV
-
       - name: Create Release
         id: create_release
         uses: zendesk/action-create-release@v1
```

Comment:
Until zendesk updates their action, warning is still gonna be generated about set-output and nodejs12